packages/f/font-freefont-ttf/files/freefont.metainfo.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>freefont</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>FreeFont</name>
+  <homepage>https://www.gnu.org/software/freefont/</homepage>
```

Comment:
Should be:
```suggestion
  <url type="homepage">https://www.gnu.org/software/freefont/</url>
```

I know you copied it off one of my faulty commits, sorry about that! Please amend the commit and force push.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url

Reply:
Done